src/ComputerPlayer.java

```diff
@@ -3,7 +3,11 @@ public class ComputerPlayer extends Player{
 	private final int MIN_POINTS = 15;
 	
 	public ComputerPlayer(){
+		//super("R2D2");
+		// The cause of the error was that the name was never defined in the constructor and 
+		// was only used to set the score. My set myName = "R2D2" like in GUIPlayer
 		super("R2D2");
+		myName = "R2D2";
```

Comment:
This might fix the bug for the computer player, but not for all players.  Since the name is being passed to the super constructor, it is better to fix it there.